WordPress/Classes/ViewRelated/Blog/Blog Dashboard/DashboardCard.swift

```diff
@@ -143,6 +151,8 @@ enum DashboardCard: String, CaseIterable {
     enum RemoteDashboardCard: String, CaseIterable {
         case todaysStats = "todays_stats"
         case posts
+        case pages
+        case activity
```

Comment:
Heads up: This change causes the dashboard cards request to fail. It will keep failing until the endpoint is updated to handle the new cards.